### DISCUSSION TITLE
How  to train a dataset of 4096X16 picture?

### DISCUSSION THREAD

**Discussion by zy486at189cn** [open-mmlab/mmclassification#1158] [Q&A]
How  to train a dataset of 4096X16 picture? The dataset's picture size is 4096X16,what model can be chosen? how to modify the configs?

**Comment by Ezra-Yu** [comment]
`4096X16` means `w = 4096, h = 16`? You can first try mobilenet. refer to [this tutorial
](https://colab.research.google.com/github/open-mmlab/mmclassification/blob/master/docs/en/tutorials/MMClassification_tools.ipynb#scrollTo=8QfM4qBeWIQh).
 pay attention to the pipeline.
